dashboard/app/models/school_info.rb

```diff
@@ -114,7 +113,13 @@ def sync_from_schools
       # Report if we are overriding a non-nil value that was originally passed in
       something_overwritten = original.map {|key, value| value && value != self[key]}.reduce {|acc, b| acc || b}
       if something_overwritten
-        Honeybadger.notify("Overwriting passed in data for new SchoolInfo", context: {original_input: original, school_id: school.id})
+        Honeybadger.notify("Overwriting passed in data for new SchoolInfo",
+          error_class: "SchoolInfo.sync_from_schools",
```

Comment:
I think you might want to specify `error_message` too, to get a more descriptive message in honeybadger.

Reply:
I fixed this. The first parameter is supposed to be the error message and the second a hash of options. Because I wasn't passing the second as an explicit hash it was treating all the args together as a single hash.